### PR TITLE
add more declarations to backend/code.d

### PR DESCRIPTION
**Pull request by WalterBright**

### CODE REVIEW

**Comment by WalterBright** [comment]
Pulling this will reduce the size/complexity of https://github.com/dlang/dmd/pull/6177

**Comment by WalterBright** [comment]
Blocking https://github.com/dlang/dmd/pull/6177

File: src/backend/code.d [hunk 1]
```diff
@@ -34,6 +38,68 @@ struct LocalSection
...
+        int segidx;                     // internal object file segment number
+        int lnameidx;                   // lname idx of segment name
+        int classidx;                   // lname idx of class name
+        uint attr;                      // segment attribute
+        targ_size_t origsize;           // original size
+        long seek;                      // seek position in output file
+        void* ledata;                   // (Ledatarec) current one we're filling in
+    }
+
+    static if (1) //ELFOBJ || MACHOBJ
```
Comment (andralex) [inline src/backend/code.d]: So... 1 or a condition? Just delete this for now and insert later if needed. We need to clean such petrified lava.

**Review by andralex — CHANGES_REQUESTED** [review]
Travis also seems to fail. cc @MartinNowak

File: src/backend/code.d (hunk 1 above)
Comment (WalterBright) [inline src/backend/code.d]: done

**Comment by andralex** [comment]
Auto-merge toggled on

**Pull request merged**